modules/electrophysiology_browser/README.md

```diff
@@ -0,0 +1,40 @@
+# Electrophysiology Browser
+
+## Purpose
+
+The Electrophysiology Browser is intended to allow users to view candidate
+electrophysiology (EEG, MEG...) sessions collected for a study.
+
+## Intended Users
+
+The primary types of users are:
+1. Electrophysiology researchers who want to know details about the inserted datasets.
+2. Site coordinators or researchers ensuring the uploaded electrophysiology data have
+been correctly inserted into LORIS.
+
+## Scope
+
+The Electrophysiology Browser displays electrophysiology datasets that have been
+inserted into LORIS from a BIDS-format collection. Derived or processed electrophysiology
+datasets can also be accessed via this module.
+
+NOT in scope at the moment:
+
+Visualization of channel signal data and electrode positions for each electrophysiology dataset. 
+(These are in development and will be added in future.)
```

Comment:
The last sentence implies that it is, in fact, within the scope of what the module should cover..

Reply:
This doesn't seem to be resolved

Reply:
I am not sure to understand that one. The visualization is in the scope of the module but no one is there to develop it so it is not there yet. Should I just remove the last two sentences and we'll add them back when the vis will be developed?

Reply:
Oh, sorry, I misread based on a previous iteration of the document..

But 'Visualization of channel signal data and electrode positions for each electrophysiology dataset.' also doesn't make sense as a sentence, so you can either update the sentence to fix the grammar or remove it..